packages/web3-providers/src/types/Firefly.ts

```diff
@@ -20,4 +20,88 @@ export namespace FireflyBaseAPI {
     export type LensResult = Result<LensAccount[]>
 
     export type VerifyTwitterResult = { error: string } | { data: string[] }
+
+    export interface UnionProfileOptions {
+        twitterId?: string
+        walletAddress?: string
+        /** e.g vitalik.lens */
+        lensHandle?: string
+        /** hex string */
+        farcasterUserName?: string
+        lensProfileId?: string
+        fid?: string
+    }
+
+    interface MediaSet {
+        original: {
+            url: string
+            mimeType: string
+        }
+    }
+    export interface LensProfile {
+        address: string
+        network: string
+        platform: string
+        source: string
+        name: string
+        handle: string
+        bio: string
+        url: string
+        profile_uri: string
+        picture: {
+            original: {
+                url: string
+                mimeType: string
+            }
+        }
+        coverPicture: MediaSet
+        stats: {
+            totalFollowers: string
+            totalFollowing: string
+            totalPosts: string
+        }
+        totalComments: string
+        totalMirrors: string
+        totalPublications: string
+        totalCollects: string
+
+        attributes: any[]
```

Comment:
Avoid any.

Reply:
We don't use this field, and I leave the docs in the comment, we can add this typing once we need in the future.